pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
@7c6f434c should `gnumake` also be added to the build inputs? Can `gnumake` build on Darwin?